pyuvdata/uvcal.py

```diff
@@ -200,6 +200,23 @@ def __init__(self):
                                          expected_type=str,
                                          required=False)
 
+        desc = ('Calfile version used in this file.')
+        self._calfile = uvp.UVParameter('calfile', form='str',
```

Comment:
I'm not sure we want to record the calfile this way, mostly because the concept of cal files is very specific to PAPER & current HERA data and it's going to go away in future HERA processing. pyuvdata is meant to be a general tool for the community, so I'm uncomfortable with having a parameter this specific and transient on the object. I suggest adding a comment to the history parameter that records the cal file. The history parameter is traditionally where code-specific parameters are recorded.